build/publish/PublishNupkgToBlobFeed.targets

```diff
@@ -22,4 +22,28 @@
                     PublishFlatContainer="false"
                     SkipCreateManifest="$(IsNotOrchestratedPublish)" />
   </Target>
+
+  <Target Name="PublishNonShipNupkgToBlobFeed" Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " >
```

Comment:
@mmitche @dagood  Do we need to have a separate invocation of PushToBlobFeed or will it read the metadata off of ItemsToPush?

Reply:
We do not want the metadata to apply to both nupkgs, but just the one.

Reply:
I don't think so.  I think you can just add the metadata to the msbuildsdkresolver item and drastically simplify this PR.

Reply:
@johnbeisner Can you just do this?

```
<ItemGroup>
  <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
</ItemGroup

<ItemGroup>
  <NupkgsForPublishing Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg">
    <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
  </NupkgsForPublishing>
</ItemGroup
```

Reply:
That's correct, it uses the `ManifestArtifactData` from each item individually.

Reply:
@mmitche
Does this work like we intend?

Reply:
@johnbeisner It should, though we'll verify once this goes in.

Reply:
Done...